tests/e2e/sample-applications/storageclass-immediate/aws-immediate.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: v1
+kind: List
+items:
+  - kind: StorageClass
+    apiVersion: storage.k8s.io/v1
+    metadata:
+      name: test-gp3-csi-immediate
+    provisioner: ebs.csi.aws.com
```

Comment:
we need to create one file for each cloud (aws, ibm, gcp and azure) right?

otherwise, virt tests can only be run in AWS

Reply:
Yes, this should just be a copy of the existing storage classes with the volumeBindingMode set to immediate. I didn't add them because I don't have all the different cluster types set up to test.

Reply:
Maybe we can automate logic here instead of creating files

The StorageClass I usually use in IBM has this annotation
```
metadata:
  annotations:
    storageclass.kubernetes.io/is-default-class: "true"
```
Does this one you are using also have it in AWS?

if yes, we could create StorageClass for tests by deepCopying result and changing its name and volumeBindingMode

Reply:
Yes, that would be nicer. It's definitely a good way to add the default class annotation too, I just didn't add it yet because the YAML templates got me to test results faster.

Reply:
Okay, I have moved this out of the YAML.